### DISCUSSION TITLE
プロキシ経由でアクセスすると無効なURLにリダイレクトされることがある

### DISCUSSION THREAD

**Discussion by YOCKOW** [increments/qiita-discussions#442] [要望・アイデア・不具合]
昨日、「[Qiita.comにプロキシサーバー経由でアクセスするときの注意点](https://qiita.com/YOCKOW/items/b257874a858a4f659b64)」という記事を書きましたが、もしかしたら不具合の可能性もあると思い直し、こちらにも投稿することにしました。

内容は標題の通りで、詳細は記事を参照いただければ幸いです。宜しくお願いいたします。

**Comment by tanaka-asami415** [comment]
@YOCKOW 
ご報告ありがとうございます🙏
恐れ入りますが、プロキシサーバー経由でのアクセスは非推奨となるため、不具合ではございません。
また、予告なくアクセス制御を行うこともございます。予めご了承いただけますと幸いです。

**Reply by YOCKOW** [reply]
ご返信ありがとうございます。
プロキシサーバー経由でのアクセスは非推奨とは知らず、申し訳ありませんでした。
その旨、当該記事にも追記しておきます。
